virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc

```diff
@@ -12,6 +12,11 @@ You can connect a virtual machine (VM) to an Open Virtual Network (OVN)-Kubernet
 
 * A localnet topology connects the secondary network to the physical underlay. This enables both east-west cluster traffic and access to services running outside the cluster, but it requires additional configuration of the underlying Open vSwitch (OVS) system on cluster nodes.
 
+[NOTE]
+====
+An OVN-Kubernetes secondary network is compatible with the xref:../../networking/multiple_networks/configuring-additional-network.adoc#compatibility-with-multi-network-policy_configuring-additional-network[multi-network policy API] which provides the `MultiNetworkPolicy` custom resource definition (CRD) to control traffic flow to and from VMs based on the `ipBlock` policy selector.
```

Comment:
```suggestion
An OVN-Kubernetes secondary network is compatible with the xref:../../networking/multiple_networks/configuring-additional-network.adoc#compatibility-with-multi-network-policy_configuring-additional-network[multi-network policy API] which provides the `MultiNetworkPolicy` custom resource definition (CRD) to control traffic flow to and from VMs using the `ipBlock` network policy peer.
```

I *think* (really not sure ...) we should call it network policy **peer** since that is its name in the API: 
https://kubernetes.io/docs/reference/kubernetes-api/policy-resources/network-policy-v1/#NetworkPolicySpec

> ingress.from ([]NetworkPolicyPeer)
> NetworkPolicyPeer describes a peer to allow traffic to/from. Only certain combinations of fields are allowed

Reply:
maybe even say "exclusively" ... 

We should be explicit that the **only** policy peer that can be used is that one.

Reply:
@maiqueb Trying to figure out the proper wording for this. Does this make sense?

```suggestion
An OVN-Kubernetes secondary network is compatible with the xref:../../networking/multiple_networks/configuring-additional-network.adoc#compatibility-with-multi-network-policy_configuring-additional-network[multi-network policy API]. You can use the `ipBlock` attribute in the `MultiNetworkPolicy` custom resource definition (CRD) to define a network policy peer to control traffic flow to and from VMs.
```

Reply:
better. 

Still I think it might be a bit confusing, because the `ipBlock` attribute is not a direct attribute of the multi network policy; it is an attribute of the ingress / egress rules belonging to the policy ... 

It's hard to get this right ...

Reply:
Another iteration
```suggestion
An OVN-Kubernetes secondary network is compatible with the xref:../../networking/multiple_networks/configuring-additional-network.adoc#compatibility-with-multi-network-policy_configuring-additional-network[multi-network policy API] which provides the `MultiNetworkPolicy` custom resource definition (CRD) to control traffic flow to and from VMs.  You can use the `ipBlock` attribute to define network policy ingress and egress rules for a specific CIDR block.
```

Reply:
@maiqueb Thoughts on the latest version above?

Reply:
Reads well enough IMO.

Thank you.